dicemix.go

```diff
@@ -196,7 +192,7 @@ func (s *Session) DiceMix(ctx context.Context, conn net.Conn, genConf GenConfirm
 		return err
 	}
 	pr := messages.PairRequest(s.Pk, s.Sk, s.commitment, unmixed, s.mcount)
-	err = s.client.send(pr, sendTimeout)
+	err = s.client.send(pr, time.Now().Add(5*time.Second))
```

Comment:
sendTimeout?

Reply:
switched to something shorter after the client side timeouts were jacked up to an hour.  no need for that on the first message sent.  could use a different constant, but it would be only used here in one place.